[Windows] SDK version not found; retargeting solution has no effect

When building the example (or testbed) app  for Windows, I get this error stating that the Windows SDK version 8.1 was not found.

A fix is suggested: either installing the required version of the SDK or retargeting the projects.
I've attempted to fix the issue by retargeting the solutions to my current Windows SDK version (>10.0), but I see the same error after doing so.

```
[ ] Project "C:\Users\matth\code\flutter-desktop-embedding\example\windows\Runner.vcxproj.metaproj" (2) is building "C:\Users\matth\code\flutter-desktop-embedding\example\windows\FlutterBuild.vcxproj" (3) on node 1 (default targets).
[ ] C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCTargets\Microsoft.Cpp.WindowsSDK.targets(46,5): error MSB8036: The Windows SDK version 8.1 was not found. Install the required version of Windows SDK or change the SDK version in the project property pages or by right-clicking the solution and selecting "Retarget solution". [C:\Users\matth\code\flutter-desktop-embedding\example\windows\FlutterBuild.vcxproj]
```

```
$ flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel master, v1.10.13-pre.12, on Microsoft Windows [Version 10.0.18362.418], locale en-US)
[√] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[√] Visual Studio - develop for Windows (Visual Studio Community 2017 15.9.11)
[√] Android Studio (version 3.2)
[√] IntelliJ IDEA Community Edition (version 2018.3)
[√] VS Code (version 1.39.2)
[√] Connected device (1 available)
```

--- a/example/windows/FlutterBuild.vcxproj
+++ b/example/windows/FlutterBuild.vcxproj
@@ -14,6 +14,7 @@
     <VCProjectVersion>15.0</VCProjectVersion>
     <ProjectGuid>{6419BF13-6ECD-4CD2-9E85-E566A1F03F8F}</ProjectGuid>
     <ProjectName>Flutter Build</ProjectName>
+    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Label="Configuration">

--- a/testbed/windows/FlutterBuild.vcxproj
+++ b/testbed/windows/FlutterBuild.vcxproj
@@ -14,6 +14,7 @@
     <VCProjectVersion>15.0</VCProjectVersion>
     <ProjectGuid>{6419BF13-6ECD-4CD2-9E85-E566A1F03F8F}</ProjectGuid>
     <ProjectName>Flutter Build</ProjectName>
+    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Label="Configuration">

